libqtile/backend/x11/window.py

```diff
@@ -1429,6 +1429,10 @@ def cmd_static(
         hook.fire("client_managed", s)
 
     def tweak_float(self, x=None, y=None, dx=0, dy=0, w=None, h=None, dw=0, dh=0):
+        # Floating is tweaked so the window
+        # cannot be fullscreen
+        if self.fullscreen:
```

Comment:
I'm not sure if this is needed but I wanted to reduce the fullscreen setter calls.